src/components/list_group/list_group.test.tsx

```diff
@@ -32,6 +32,13 @@ const someListItems: EuiListGroupItemProps[] = [
     onClick: (e) => {
       console.log('Visualize clicked', e);
     },
+    toolTipProps: {
+      children: <button />,
+      delay: 'regular',
+      id: '12345',
+      position: 'top',
+      title: 'Title of record',
+    },
```

Comment:
This is a facet of incorrect typing on `toolTipProps`, `children` should not be required when passed. I also would not expect most consumers to pass in a custom ID
```suggestion
    toolTipProps: {
      delay: 'regular',
      position: 'top',
      title: 'Title of record',
    },
```